source/modules/collection-api/transfer/overview.md.erb

```diff
@@ -1,19 +1,16 @@
-## Items Transfers
+## Item Transfers
 
-The following functions will be used to relay item (*e.g. ticket*) transfers between users (*e.g. accounts*) that happen within your platform:
+You can provide transfer-related activity via the REST API. The `/transfer` endpoint supports POST and PUT operations with the following required arguments:
```

Comment:
Any reason why this is the only end-point in singular form (ie transfer, not transfers; whereas the others are accounts, sales, etc)?

Reply:
ah, should be plural for consistency

Reply:
Actually, noticed that the `/sale` endpoint is also incorrect in this regard. Do you guys have a preference for plural vs singular universally? (Tbh I actually think singular might be better?)

cc @diguid

Reply:
As each request has references to a single object, I think I prefer singular.